components/core/src/clp/streaming_archive/ArchiveMetadata.hpp

```diff
@@ -2,13 +2,19 @@
 #define CLP_STREAMING_ARCHIVE_ARCHIVEMETADATA_HPP
 
 #include <cstdint>
+#include <string_view>
 
 #include "../Defs.h"
+#include "../ffi/encoding_methods.hpp"
 #include "../FileReader.hpp"
 #include "../FileWriter.hpp"
 #include "Constants.hpp"
+#include "msgpack.hpp"
```

Comment:
```suggestion
#include <msgpack.hpp>
```

Should use `<>` for library includes.